mkosi/config.py

```diff
@@ -2160,7 +2163,7 @@ def parse_ini(path: Path, only_sections: Collection[str] = ()) -> Iterator[tuple
         yield section, "", ""
 
 
-PE_ADDON_SETTINGS = (
+PE_ADDON_SETTINGS: list[ConfigSetting[Any]] = [
```

Comment:
This can be `Any` as well I think same elsewhere

Reply:
This is any. And using object here breaks because the members are not homogeneous and inference goes haywire